File: Python/account.py
class Account:
    id         =  int
    name       =  str
    document   =  str
    email      =  str
    password   =  str

    def __init__(self, name, document):
        self.name       = name
        self.document   = document

    def printUserData(self):
        print("Name account: " + self.name)
        print("Identification document: " + self.document)
  

File: JS/Account.js
class Account
{
    constructor(name, document)
    {
        this.id;
        this.name = name;
        this.document = document;
        this.email;
        this.password;
    }   
    
    printUserData()
    {
        console.log("Nombre usuario: " + this.name);
        console.log("Identificación usuario: " + this.document);
    }
}



    